bt_editor/mainwindow.cpp

```diff
@@ -531,6 +532,16 @@ void showVec(vector<T> v) {
     cout << endl;
 }
 
+string without_number(const string& str) {
```

Comment:
um... this function name and parameter needs to be more meaningful... maybe `leave_only_alphabet`?